package.json

```diff
@@ -174,6 +175,7 @@
     "lusca": "^1.4.1",
     "material-ui": "^0.16.4",
     "moment": "^2.13.0",
+    "momentjs": "^2.0.0",
```

Comment:
yep, it's just leftovers in package.json, I am using moment, just need to clean up package.json